app/views/shared/_translations.html.erb

```diff
@@ -1,7 +1,6 @@
-<% if @content_item.available_translations.any? %>
+<% if @content_item.available_translations.any? && @content_item.available_translations.length > 1 %>
```

Comment:
I think because we don't want to render a translation navigation item for only one translation.. there should always be a link to the current language the page is being displayed in?